cypress/e2e/create-edit-delete-mission.cy.ts

```diff
@@ -0,0 +1,162 @@
+            }
+        })
+    })
+})
+describe("When creating a mission :", () => {
+    beforeEach(() => {
+        cy.netlifyLogin();
+    });
+    it("Should fill the form, submit, show a notification and redirect to dashboard", () => {
+        cy.visit("/dashboard/mission/create/");
+        cy.get("input[id=mission_title]").type("Mission 1");
+        cy.get("input[id=mission_start-date]").type("2020-01-01");
+        cy.get("input[id=mission_end-date]").type("2020-01-02");
+        cy.get("input[id=mission_details-unitOfWorkPrice]").type("100");
+        cy.get("input[id=mission_consultant]").type("Consultant 1");
+        cy.get("input[id=mission_consultant-email]").type("consultant@email.com");
+        cy.get("input[id=consultantCompanyName]").type("Consultant Company");
+        cy.get("input[id=consultantCompanyAddress]").type("1 rue de la Consultation");
+        cy.get("input[id=consultantCompanyPhone]").type("0123456789");
+        cy.get("input[id=consultantCompanySiren]").type("123456789");
+        cy.get("input[id=consultantCompanyTradeAndCompaniesRegisterCity]").type("Paris");
+        cy.get("input[id=consultantCompanyVatNumber]").type("54321");
+        cy.get("input[id=mission_consultant-unitOfWorkPrice]").type("100");
+        cy.get("input[id=consultantCompanyHolder]").type("Consultant Company");
+        cy.get("input[id=consultantCompanySwift]").type("123456789");
+        cy.get("input[id=consultantCompanyIban]").type("123456789");
+        cy.get("input[id=consultantCompanyDomiciliation]").type("Banque 1");
+        cy.get("input[id=consultantCompanyAgency]").type("Agence 1");
+        cy.get("input[id=mission_client-ref]").type("REF01");
+        cy.get("input[id=missionClientEmail]").type("client@email.com");
+        cy.get("input[id=mission_client_name]").type("Client 1");
+        cy.get("input[id=mission_client_address]").type("1 rue du Client");
+        cy.get("input[id=mission_client_phone]").type("0123456789");
+        cy.get("input[id=mission_client_siren]").type("123456789");
+        cy.get("input[id=mission_client_trade-and-companies-register-city]").type("Paris");
+        cy.get("input[id=mission_client_vat-number]").type("54321");
+        cy.get("input[id=mission_details-paymentMode]").type("Virement");
+        cy.get("input[type=submit]").click();
+        cy.url().then(
+            (url) => expect(url.endsWith("/dashboard/mission/all")).to.be.true
+        );
+        cy.get(".notifications__item").should("exist");
+    })
+    it("Should display the mission in the list", () => {
+        cy.visit("/dashboard/mission/all");
+        cy.wait(2000)
+        var ok = false
+        cy.get(".d-block").each(($el) => {
+            if ($el.text() == "Consultant 1 Client 1 | 01/01/20 ") {
+                ok = true
+                return
+            }
```

Comment:
Few points that will apply for multiple blocks in these tests as they are really not up to standards :

- Don't use `var` it's a relic of the begining of JS, in modern JS we use `let` and `const`
- The formatting doesn't match with the rest of the file, there is no semicolon at the end of those lines
- you shouldn't need to add a `cy.wait(2000)` here, Cypress already wait up to 5000 ms on every command made before failling
- Worst, they are not testing the element content properly (read "in the cypress way", see suggestion). 

In order to write a good test you should use the "Arrange - Act - Assert" pattern and if you have to write `throw <your error>` then something is not right. There is two key components of a testing environment, a runner and an assertion library, the runner runs the tests and the assertion library checks if returned values are correct or not.
Cypress is it's own test runner AND assertion library (https://docs.cypress.io/guides/references/assertions)

Idk if these tests were generated using Copilot or not, it's not a problem if you did but make sure that their quality is up to standards before submitting a PR.

```suggestion
        cy.get(".d-block").each(($el) => {
            cy.wrap($el).should("have.text",  "Consultant 1 Client 1 | 01/01/20");
        });
```